spec/current_players_spec.rb

```diff
@@ -48,5 +48,15 @@ def make_a_player(marker)
 
             expect(new_board_spaces).to eq ["", "", "", "", "", "", "", "", "O"]
         end
+
+        def handle_turn(marker)
+            TicTacToe::Players.new(marker)
+        end
+        context "when player is taking a turn" do
+            it "gets the position the player selected" do
+                position = handle_turn(6)
```

Comment:
So since handle_turn doesn't actually handle a turn, it just news up a new player that puts "6" on squares on the board,
It may not be doing what you want.

It's true that `position.marker` is probably "6", but what isn't true is that you haven't marked the board. 
you named your player `position` and then asked for that player's marker rather than creating a player with a marker like "X" and then telling that player to `.mark(the_current_board, on_space_six)`

Then you'd expect the board[6] you get from `player.mark(empty_board, 6)` to equal "X"

It's all a matter of using words to help you.

Use Given , When , Then language to help organize the test and clarify your thoughts on what this is supposed to be doing :)